.github/workflows/kubeapps-general.yaml

```diff
@@ -617,7 +617,7 @@ jobs:
   sync_chart_from_bitnami:
     needs:
       - setup
-    if: needs.setup.outputs.running_on_main == 'true'
+    if: needs.setup.outputs.running_on_main == 'true' || needs.setup.outputs.running_on_tag == 'true'
```

Comment:
I'm still not clear in my mind why we want to sync the bitnami chart each time a PR targets main? Wouldn't we only want to do so if there isn't a PR already open to sync the bitnami chart into our main? (given that the one PR can be recreated or just have latest changes merged etc?)

Thanks either way!